src/_posts/platform/getting-started/2000-01-01-how-to-migrate-from-heroku.md

```diff
@@ -5,43 +5,78 @@ tags: heroku tutorial
 index: 11
 ---
 
-Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). It's very easy to understand and you shouldn't have any problem migrating your app from Heroku to Scalingo.
+Scalingo is a Platform as a Service [highly compatible with Heroku]({% post_url platform/getting-started/2000-01-01-heroku-compatibility %}). This page is here to help your migrate from Heroku to Scalingo as easir as possible.
 
-It is so easy that you can use [heroku2scalingo](https://github.com/Scalingo/heroku2scalingo/releases/latest), a tool to migrate a Heroku app to Scalingo.
+## Requirements
 
-## heroku2scalingo
+To be able to follow this tutorial, we assume that you have:
 
-### Usage
+- A Scalingo account
+- The Scalingo [CLI]({% post_url platform/cli/2000-01-01-start %}) installed
+- A local copy of the repository containing your app’s code
+
+Please ensure you are running all commands above from the root of the repository containing your app’s code. 
 
 ```bash
-./heroku2scalingo <herokuAppName>
+$ cd <app directory>
 ```
 
-It works by performing the following operations in the same order as below:
+Note: All operations described below can also be done through our web dashboard (instead of the command line tool). 
+
+## Migration Steps
 
-* Authentication to Scalingo
-* Authentication to Heroku API
-* Creation of Scalingo app
-* Get/Set environment variables
-* `git clone` your Heroku app repository
-* `git push scalingo master` -> Auto-deployment using the `Procfile`
+### Creating your Scalingo app
 
-### Notes
+- Start by login into the Scalingo platform using our `cli` :
 
-Other advanced operations are planned, for more informations take a look at [the GitHub project](https://github.com/Scalingo/heroku2scalingo/#todo).
+```bash
+$ scalingo login
+```
 
-### Release
+- You can now create your app : 
+
+```bash
+# Replace <my-app> by your actual app name
+$ scalingo create <my-app>
```

Comment:
In all the documentation we use `my-app` without the `<` and `>` characters. Hence I would keep this page consistent with the remaining of the doc pages:

```suggestion
# Replace my-app by your actual app name
$ scalingo create my-app
```

Reply:
Oups that's my doing